opentelemetry-api/src/opentelemetry/util/_providers.py

```diff
@@ -30,23 +36,51 @@
 def _load_provider(
     provider_environment_variable: str, provider: str
 ) -> Provider:
+
     try:
-        entry_point = next(
-            iter_entry_points(
-                f"opentelemetry_{provider}",
+
+        # FIXME remove when support for 3.9 is dropped.
+        if version_info.minor <= 9:
+
+            provider_name = cast(
```

Comment:
Why is this needed for only versions <= 3.9?

Reply:
Because `entry_points` has 2 different interfaces, one for 3.7 and 3.10 and one for 3.8 and 3.9.